pdf2image/pdf2image.py

```diff
@@ -221,18 +221,17 @@ def _page_count(pdf_path, userpw=None, poppler_path=None):
         env = os.environ.copy()
         if poppler_path is not None:
             env["LD_LIBRARY_PATH"] = poppler_path + ":" + env.get("LD_LIBRARY_PATH", "")
-        proc = Popen(command, env=env, stdout=PIPE, stderr=PIPE)
+        proc = Popen(command, env=env, stdout=PIPE, stderr=PIPE, universal_newlines=True)
 
         out, err = proc.communicate()
     except:
         raise PDFInfoNotInstalledError('Unable to get page count. Is poppler installed and in PATH?')
 
     try:
         # This will throw if we are unable to get page count
-        return int(re.search(r'Pages:\s+(\d+)', out.decode("utf8", "ignore")).group(1))
+        return int(re.search(r'Pages:\s+(\d+)', out).group(1))
     except:
-        raise PDFPageCountError('Unable to get page count. %s' % err.decode("utf8", "ignore"))
-
+        raise PDFPageCountError('Unable to get page count. %s' % out)
```

Comment:
Any reason why we do not print the stderr here instead of the stdout?

Reply:
nop my mistake that should be err, i will correct that